extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -6,7 +6,7 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (googleSlots, helper, doc, log, window) {
+], function (googleSlots, recovery, doc, log, window) {
```

Comment:
do we need to load `recovery` on mobile if we don't have recovery on mobile? Maybe we can do some tricks with `require.optional` and remove `'//extensions/wikia/ARecoveryEngine/js/helper.js',` from `mobile_base_ads` package?